package.json

```diff
@@ -4,6 +4,6 @@
     "elm-test": "^0.18.12"
   },
   "scripts": {
-    "test": "elm-make --yes --docs documentation.json && elm-test"
+    "test": "elm-make --yes --docs documentation.json && elm-package diff && elm-test"
```

Comment:
why add diff here but not test the output?

Reply:
This was added just so you would see it in the output and could notice if something was not what you expected.  It's not meant to fail if there are API changes.